### PR TITLE
Update Cors Example to RC1

### PR DESCRIPTION
**Pull request by robbert229**
Please review @MikeWasson

### CODE REVIEW

**Comment by MikeWasson** [comment]
Looks great, thanks!

**Comment by danroth27** [comment]
@kichalla Can you take a look?

File: aspnet/security/cors/sample/src/CorsExample2/Startup.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿using Microsoft.AspNet.Builder;
```
Comment (kichalla) [inline aspnet/security/cors/sample/src/CorsExample2/Startup.cs]: All the samples here seem to use Cors middleware only...this is not very common thing to do as far as I know...Cors can be used without the CorsMiddleware...i.e MVC middleware has support for Cors (which it does through the Cors authorization filter)...I think we should have more samples using this...

**Comment by kichalla** [comment]
:shipit:

File: aspnet/security/cors/sample/src/CorsExample2/Startup.cs (hunk 1 above)
Comment (danroth27) [inline aspnet/security/cors/sample/src/CorsExample2/Startup.cs]: This CORS doc is in the ASP.NET repo, so the content here should be generic to ASP.NET and not specific to MVC. The MVC CORS support is documented here: http://docs.asp.net/projects/mvc/en/latest/security/cors-policy.html

**Comment by danroth27** [comment]
Thanks! 9276c3b78cab58de5eeeb1513750db6968011d70

**Pull request closed without merging**